doc/source/ecoharmonogram_pl.md

```diff
@@ -27,8 +78,32 @@ waste_collection_schedule:
         street: Wyki
 ```
 
-Worth to mention that street and house_number are optional parameters if your city doesn't require one.
+### Example with community and street number
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: ecoharmonogram_pl
+      args:
+        community: "108"
```

Comment:
I feel that users should not know some magic ID. This should be probably set when the user uses Gdańsk or Niemce as input for town.

Reply:
That makes sense, however there is no dictionary (I'm aware of) that would allow translating literals in community ID. There is an updated chapter in [ecoharmonogram_pl.md](https://github.com/mampfes/hacs_waste_collection_schedule/pull/405/commits/3f530c8692ca3e07cc6e0a56b48051f06cceb34b) file that mentions how to obtain that ID.

Reply:
On that note, I as no difference here with using explicit strings, like ["wie", "jedn" in sides_matcher:](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/custom_components/waste_collection_schedule/waste_collection_schedule/source/ecoharmonogram_pl.py#L10)

Reply:
Resolving conflicts at the current state would mean more than re-baselining from `master` and adding the changes again, so I will withdraw this PR.